index.html

```diff
@@ -187,25 +218,33 @@ <h2 class="title__create">Erstelle deine eigene Quizfrage</h2>
             <input class="tag1" type="text" placeholder="tag 4" />
           </div>
 
-          <button class="btn__submit">Eintragen</button>
+          <button type="button" class="btn__submit">Eintragen</button>
         </form>
       </section>
     </main>
     <footer class="footer">
       <nav class="footer__nav">
-        <button id="btn-home">
-          <img src="/img/Home.svg" width="70" alt="" />
+        <button class="nav__buttons" id="btn-home">
+          <img src="/img/Home.svg" width="60" alt="" />
+        </button>
+        <button class="nav__buttons" id="btn-bookmark">
+          <img src="/img/bookmark.svg" width="50" alt="" />
         </button>
-        <button id="btn-bookmark">
-          <img src="/img/bookmark.svg" width="45" alt="" />
+        <button class="nav__buttons" id="btn-create">
+          <img src="/img/book-plus.svg" width="60" alt="" />
         </button>
-        <button id="btn-create">
-          <img src="/img/book-plus.svg" width="70" alt="" />
+        <button class="nav__buttons" href="#">
+          <img src="/img/human-male-board.svg" width="60" alt="" />
         </button>
-        <button href="#">
-          <img src="/img/human-male-board.svg" width="70" alt="" />
+        <button
+          class="nav__buttons"
+          id="btn-darkmode"
+          onclick="darkFunctionBundle()"
+        >
+          <img src="/img/night-mode.svg" width="60" alt="" />
         </button>
       </nav>
     </footer>
+    <script src="/main.js"></script>
```

Comment:
Would be better to add it in the `head` with `<script src="/main.js" defer></script>`